src/genreclassutil.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import unicode_literals
+import re
+import subprocess
+import urllib.request
+import numpy as np
+import librosa
+import librosa.display
+import matplotlib.pyplot as plt
+import os
+from pydub import AudioSegment
+
```

Comment:
There should be two lines of space between import statements, function, classes. One line of space between methods in classes.